packages/browser/src/index.ts

```diff
@@ -25,4 +44,36 @@ export default class LogtoClient {
   constructor(logtoConfig: LogtoConfig) {
     this.logtoConfig = logtoConfig;
   }
+
+  public async signInWithBrowser(redirectUri: string) {
```

Comment:
Do we need to specificly call this signIn`WithBrowser`?

Reply:
Follow SDK Convention https://www.notion.so/silverhand/signIn-bb0b307176f644be8f3e763189f00159

<img width="743" alt="image" src="https://user-images.githubusercontent.com/10594507/153561186-1ee01948-6a45-4a22-8716-7bee7442b0b8.png">

Reply:
we are already in a browser, so no need to append `WithBrowser`?

Reply:
Simplified.